src/components/SocialButtons.js

```diff
@@ -0,0 +1,249 @@
+import React, { Component } from "react";
+import {
+  EmailShareButton,
```

Comment:
See comment below, if we decide use just that six services, do you need to include all these? It seems we will never use most of services...

Reply:
I think we should include all of them and in a separate ticket set up a configuration option in the admin area to let the stakeholders choose which social media to allow. I just picked 6 for now to demonstrate.

Reply:
I don't think we should include all of them, as I said earlier, some services we will never use, so why include the codes we will never use?

I do agree we can have another ticket for configuration option.

Reply:
My apologies, I thought your previous message was asking for my opinion. I don't feel strongly either way about this, it was just my opinion. I just pushed changes that removes all the lesser known networks, so the options are reduced by about half.